inst/extdata/level-of-service-table-long.csv

```diff
@@ -15,43 +19,56 @@ Speed (85th kph),Speed Limit (mph),AADT,infrastructure,level_of_service
 30 kph to 50 kph,20 mph,4000+,Painted Cycle Lane,1
 30 kph to 50 kph,20 mph,4000+,Segregated Track (narrow),2
 30 kph to 50 kph,20 mph,4000+,Segregated Track (wide),3
-50 kph to 65 kph,30 mph,0 to 1999,Mixed Traffic Street,2
-50 kph to 65 kph,30 mph,0 to 1999,Off road cycleway,3
-50 kph to 65 kph,30 mph,0 to 1999,Painted Cycle Lane,2
-50 kph to 65 kph,30 mph,0 to 1999,Segregated Track (narrow),2
-50 kph to 65 kph,30 mph,0 to 1999,Segregated Track (wide),2
-50 kph to 65 kph,30 mph,2000 to 3999,Mixed Traffic Street,1
+50 kph to 65 kph,30 mph,0 to 999,Mixed Traffic Street,2
+50 kph to 65 kph,30 mph,0 to 999,Off road cycleway,3
+50 kph to 65 kph,30 mph,0 to 999,Painted Cycle Lane,2
+50 kph to 65 kph,30 mph,0 to 999,Segregated Track (narrow),2
+50 kph to 65 kph,30 mph,0 to 999,Segregated Track (wide),2
+50 kph to 65 kph,30 mph,1000 to 1999,Mixed Traffic Street,1
+50 kph to 65 kph,30 mph,1000 to 1999,Off road cycleway,3
+50 kph to 65 kph,30 mph,1000 to 1999,Painted Cycle Lane,1
+50 kph to 65 kph,30 mph,1000 to 1999,Segregated Track (narrow),2
+50 kph to 65 kph,30 mph,1000 to 1999,Segregated Track (wide),2
 50 kph to 65 kph,30 mph,2000 to 3999,Mixed Traffic Street,0
 50 kph to 65 kph,30 mph,2000 to 3999,Off road cycleway,3
 50 kph to 65 kph,30 mph,2000 to 3999,Painted Cycle Lane,1
-50 kph to 65 kph,30 mph,2000 to 3999,Segregated Track (narrow),2
+50 kph to 65 kph,30 mph,2000 to 3999,Segregated Track (narrow),1
```

Comment:
Good fix, although 30 mph is now 30-50 kph I think.

![image](https://github.com/user-attachments/assets/d62606c6-6140-4576-bddc-cddb3c1cb8a4)